data/nixpkgs.json

```diff
@@ -1,12 +1,12 @@
 {
-  "branch": "release-21.05",
+  "branch": "nixos-unstable-small",
```

Comment:
what's the difference between this, unstable & master?

Reply:
I think this wiki page is the best resource we have ATM https://nixos.wiki/wiki/Nix_channels#The_official_channels. Short but descriptive enough.

Reply:
Thanks! I would default to a non `-small`, since `niv` seems to be used by a lot of beginners that might get surprised if they have to rebuild packages from source; also this is what we've done so far. Wdyt?

Reply:
right though I agree with the choise of "nixos-unstable" as a default since that's what most users I can think of would be looking for.

Reply:
No strong opinion about -small. Just wanted to point out the difference between branches that are before and those that are after hydra.